### PR TITLE
updating details on EOL

### PR DESCRIPTION
**Pull request by cybermax-008**
After a customer Q/A on EOL, i found two points that could be added to the existing help article. The points were discussed over here: https://aiven-io.slack.com/archives/C01UBB57XFS/p1682621996912009

### CODE REVIEW

File: docs/platform/reference/eol-for-major-versions.rst [hunk 1]
```diff
@@ -195,6 +189,18 @@ When Aiven defines the EOL date for a service major version,
...
    the month of the EOL date, the cadence shifts to weekly reminders.
 
+What happens to the outdated services beyond EOL?
+-------------------------------------------------
+
+Continued use of outdated services means that they no longer offer our
+customers the level of protection their business needs. Therefore, by
+following the upstream project's EOL schedule, we ensure that Aiven
+services are always running on supported versions of the open source
+software. The following two scenarios will be observed beyond EOL:
```
Comment (staceysalamon-aiven) [inline docs/platform/reference/eol-for-major-versions.rst:199]:
```suggestion
software. After the EOL date:
```

File: docs/platform/reference/eol-for-major-versions.rst [hunk 2]
```diff
@@ -195,6 +189,18 @@ When Aiven defines the EOL date for a service major version,
...
+
+1. If the service is powered-on, it will be auto-upgraded to the latest version.
```
Comment (staceysalamon-aiven) [inline docs/platform/reference/eol-for-major-versions.rst:201]:
```suggestion
1. If the service is powered on, it's auto-upgraded to the latest version.
```

File: docs/platform/reference/eol-for-major-versions.rst [hunk 3]
```diff
@@ -195,6 +189,18 @@ When Aiven defines the EOL date for a service major version,
...
+2. If the service is powered-off, it will be deleted.
```
Comment (staceysalamon-aiven) [inline docs/platform/reference/eol-for-major-versions.rst:202]:
```suggestion
2. If the service is powered off, it's deleted.
```

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]

File: docs/platform/reference/eol-for-major-versions.rst (hunk 3 above)
Comment (cybermax-008) [reply docs/platform/reference/eol-for-major-versions.rst]: Thanks for the suggestions. It has been edited as such.

**Comment by staceysalamon-aiven** [comment]
The EOL article looks fine, but there appear to be other commits added here that added other files not related to this PR?

**Comment by staceysalamon-aiven** [comment]
> > The EOL article looks fine, but there appear to be other commits added here that added other files not related to this PR?
> 
> Yes, not sure how it got added! Can i close this PR and create a new one ?

That works, too, yeah! Feel free to tag me in the new PR and I'll get it approved/published. :)

**Comment by cybermax-008** [comment]
> > > The EOL article looks fine, but there appear to be other commits added here that added other files not related to this PR?
> > 
> > 
> > Yes, not sure how it got added! Can i close this PR and create a new one ?
> 
> That works, too, yeah! Feel free to tag me in the new PR and I'll get it approved/published. :)

Added a new PR - https://github.com/aiven/devportal/pull/1906

**Pull request closed without merging**